### DISCUSSION TITLE
Can I make TST create only two levels of tree?

### DISCUSSION THREAD

**Discussion by evilenzo** [piroor/treestyletab#3396] [Q&A] [answered]
I mean, I don't want the tree to be of height more than N cause I feel uncomfortable...

**Comment by irvinm** [comment, marked as answer]
I am assuming you mean you don't want the tree to be indented more than 2 levels.

![image](https://github.com/piroor/treestyletab/assets/979729/f2fda34a-776c-4b8f-b9d3-507a50255265)
